tap_github_search/tap.py

```diff
@@ -13,9 +14,14 @@ class TapGitHubSearch(TapGitHub):
     name = "tap-github-search"
 
     def discover_streams(self) -> list[Stream]:
+        search_cfg_b64 = os.environ.get("TAP_GITHUB_SEARCH_SEARCH_B64")
         search_cfg = os.environ.get("GITHUB_SEARCH_CONFIG")
+        
+        if search_cfg_b64:
+            search_cfg = base64.b64decode(search_cfg_b64).decode("utf-8")
```

Comment:
Maybe add a log what the value is after decoding, to make it easier to debug things, as the env variable section will not be very useful if it's in base64.